tests/scalability/generate_brac_data/README.md

```diff
@@ -0,0 +1,46 @@
+# Brac Config Data Generator Script (first draft)
+
+## Introduction
+
+This script is to generate Brac config test data.
+
+Initially it only creates places, persons, users and reports (pregnancy, assessment and assessment follow up). Taking into account hierarchies.
+
+Changes where made in /test/factories/ and /test/scalability/
+Basically added new folders:
+/test/factories/brac and /test/scalability/generate_brac_data
+
+## Requirements
+
+- NodeJS
+- a target medic instance with brac config loaded
+
+
+## Steps to execute it
+1. Clone cht-core repo.
+2. Run npm ci, npm install and npm install @faker-js/faker --save-dev in cht-core root directory.
+3. Go to `/tests/scalability/generate_brac_data` and edit the `data_directory` attribute in `data-config.json` doc.
```

Comment:
Can the data directory be passed as a parameter to the `generate-brac-data.js` script, and avoid needing to edit paths? This way we won't have people who generate this data accidentally commit the changes.